### ISSUE TITLE
No accessible elements in the favourite list

### ISSUE THREAD

**Issue by TyflosAccessibleSoftware** [iterate-ch/cyberduck#16247]
**Describe the bug**
The elements in the list of favourite sites are not accessible for VoiceOver
Each element in a row are identified by VoiceOver as two images without any accessible text and an empty cell.
VoiceOver is the screen reader for MacOS to help blind users to use a Macbook

**To Reproduce**
1. Open Cyberduck and the list of favourite sites should appear
2. Run VoiceOver pressing command+F5
3. Press down arrow key or up arrow key to move the cursor in the Favourite sites list and VoiceOver will speak "Image, image" in all rows

**Expected behavior**
VoiceOver should read the title of the favorite site in each row instead of "Image, image" 
Also, each row should be identified as a button instead of a text in a row

Tip: apply the role button to the row and use accessibilityLabel to set an accessible text for a UI element
In MacOS and iOS the role is named as accessibilityTrait

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 14.6

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**
Now the list of favurite sites is not usable for blind users. I have to use the items in the Favourite menu bar to choose an element

**Comment by dkocher** [comment]
Duplicate for #10853.
